src/pages/guides/using/marketo/marketo-observability-data-stream-setup.md

```diff
@@ -228,12 +228,12 @@ the processing results could potentially be reported across multiple events.
         "quota": 65432100,
         "queue": {
             "high": {
-                "lagSeconds": "2023-08-14T17:31:00Z",
+                "lagSeconds": 15,
```

Comment:
is the definition still the same for lagSeconds as per in the doc [here](https://developer.adobe.com/events/docs/guides/using/marketo/marketo-observability-data-stream-setup/#data-field-definitions)?

Reply:
@abhupadh yup, the change should more accurately reflect that definition